site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -158,8 +158,11 @@ export const WorkspaceReadyPage: FC<WorkspaceReadyPageProps> = ({
 	const cancelBuildMutation = useMutation(cancelBuild(workspace, queryClient));
 
 	// Build Timings. Fetch build timings only when the build job is completed.
+	const readyAgents = workspace.latest_build.resources
+		.flatMap((r) => r.agents)
+		.filter((a) => a && a.lifecycle_state !== "starting");
 	const timingsQuery = useQuery({
-		...workspaceBuildTimings(workspace.latest_build.id),
+		...workspaceBuildTimings(workspace.latest_build.id, readyAgents.length),
```

Comment:
What do we show when an agent is not yet ready?

Reply:
A pending state. 

![381571574-847efb7c-22fc-4e0b-8b8d-e997804a1066](https://github.com/user-attachments/assets/74fb6ad0-ce4b-467a-a612-edff8bdfaf59)